CreatubblesAPIClient.podspec

```diff
@@ -58,7 +58,7 @@ Pod::Spec.new do |s|
   #  the deployment target. You can optionally include the target after the platform.
   #
 
-  s.platform      = :ios, "8.0"
```

Comment:
I decided to bump up the deployment target to 9.0, but we can move back to 8.0 if needed